2020-03-17 16:36:04,930 INFO: Microblog startup [in C:\Users\Alec\Documents\Development\FlaskStuff\microblog\app\__init__.py:52]
2020-03-17 16:39:24,684 INFO: Microblog startup [in C:\Users\Alec\Documents\Development\FlaskStuff\microblog\app\__init__.py:52]
2020-03-17 16:40:45,211 INFO: Microblog startup [in C:\Users\Alec\Documents\Development\FlaskStuff\microblog\app\__init__.py:52]
2020-03-17 16:41:56,887 INFO: Microblog startup [in C:\Users\Alec\Documents\Development\FlaskStuff\microblog\app\__init__.py:52]
2020-03-17 16:45:22,535 INFO: Microblog startup [in C:\Users\Alec\Documents\Development\FlaskStuff\microblog\app\__init__.py:52]
2020-03-17 16:48:13,719 INFO: Microblog startup [in C:\Users\Alec\Documents\Development\FlaskStuff\microblog\app\__init__.py:52]
2020-03-17 16:48:55,855 INFO: Microblog startup [in C:\Users\Alec\Documents\Development\FlaskStuff\microblog\app\__init__.py:52]
2020-03-17 16:58:56,111 INFO: Microblog startup [in C:\Users\Alec\Documents\Development\FlaskStuff\microblog\app\__init__.py:52]
2020-03-17 17:04:24,692 INFO: Microblog startup [in C:\Users\Alec\Documents\Development\FlaskStuff\microblog\app\__init__.py:52]
2020-03-17 17:05:18,891 INFO: Microblog startup [in C:\Users\Alec\Documents\Development\FlaskStuff\microblog\app\__init__.py:52]
2020-03-17 17:06:13,677 INFO: Microblog startup [in C:\Users\Alec\Documents\Development\FlaskStuff\microblog\app\__init__.py:52]
2020-03-17 17:06:47,725 INFO: Microblog startup [in C:\Users\Alec\Documents\Development\FlaskStuff\microblog\app\__init__.py:52]
2020-03-17 17:07:10,915 INFO: Microblog startup [in C:\Users\Alec\Documents\Development\FlaskStuff\microblog\app\__init__.py:52]
2020-03-17 17:07:44,262 INFO: Microblog startup [in C:\Users\Alec\Documents\Development\FlaskStuff\microblog\app\__init__.py:52]
2020-03-17 17:08:23,681 INFO: Microblog startup [in C:\Users\Alec\Documents\Development\FlaskStuff\microblog\app\__init__.py:52]
2020-03-17 17:08:55,311 INFO: Microblog startup [in C:\Users\Alec\Documents\Development\FlaskStuff\microblog\app\__init__.py:52]
2020-03-17 17:09:34,853 INFO: Microblog startup [in C:\Users\Alec\Documents\Development\FlaskStuff\microblog\app\__init__.py:52]
2020-03-17 17:11:58,051 INFO: Microblog startup [in C:\Users\Alec\Documents\Development\FlaskStuff\microblog\app\__init__.py:52]
2020-03-17 17:12:02,553 ERROR: Exception on /translate [POST] [in c:\users\alec\documents\development\flaskstuff\microblog\venv\lib\site-packages\flask\app.py:1890]
Traceback (most recent call last):
  File "c:\users\alec\documents\development\flaskstuff\microblog\venv\lib\site-packages\flask\app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "c:\users\alec\documents\development\flaskstuff\microblog\venv\lib\site-packages\flask\app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "c:\users\alec\documents\development\flaskstuff\microblog\venv\lib\site-packages\flask\app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "c:\users\alec\documents\development\flaskstuff\microblog\venv\lib\site-packages\flask\_compat.py", line 39, in reraise
    raise value
  File "c:\users\alec\documents\development\flaskstuff\microblog\venv\lib\site-packages\flask\app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "c:\users\alec\documents\development\flaskstuff\microblog\venv\lib\site-packages\flask\app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "c:\users\alec\documents\development\flaskstuff\microblog\venv\lib\site-packages\flask_login\utils.py", line 272, in decorated_view
    return func(*args, **kwargs)
  File "C:\Users\Alec\Documents\Development\FlaskStuff\microblog\app\routes.py", line 189, in translate_text
    return jsonify({'text': translate(request.form['text_input'],
  File "C:\Users\Alec\Documents\Development\FlaskStuff\microblog\app\translate.py", line 21, in translate
    result = r[0]['translations'][0]['text']
KeyError: 0
2020-03-17 17:13:48,762 INFO: Microblog startup [in C:\Users\Alec\Documents\Development\FlaskStuff\microblog\app\__init__.py:52]
2020-03-17 17:13:53,648 ERROR: Exception on /translate [POST] [in c:\users\alec\documents\development\flaskstuff\microblog\venv\lib\site-packages\flask\app.py:1890]
Traceback (most recent call last):
  File "c:\users\alec\documents\development\flaskstuff\microblog\venv\lib\site-packages\flask\app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "c:\users\alec\documents\development\flaskstuff\microblog\venv\lib\site-packages\flask\app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "c:\users\alec\documents\development\flaskstuff\microblog\venv\lib\site-packages\flask\app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "c:\users\alec\documents\development\flaskstuff\microblog\venv\lib\site-packages\flask\_compat.py", line 39, in reraise
    raise value
  File "c:\users\alec\documents\development\flaskstuff\microblog\venv\lib\site-packages\flask\app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "c:\users\alec\documents\development\flaskstuff\microblog\venv\lib\site-packages\flask\app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "c:\users\alec\documents\development\flaskstuff\microblog\venv\lib\site-packages\flask_login\utils.py", line 272, in decorated_view
    return func(*args, **kwargs)
  File "C:\Users\Alec\Documents\Development\FlaskStuff\microblog\app\routes.py", line 189, in translate_text
    return jsonify(translate(request.form['text_input'],
  File "C:\Users\Alec\Documents\Development\FlaskStuff\microblog\app\translate.py", line 21, in translate
    result = r[0]['translations'][0]['text']
KeyError: 0
2020-03-17 17:16:55,475 INFO: Microblog startup [in C:\Users\Alec\Documents\Development\FlaskStuff\microblog\app\__init__.py:52]
2020-03-17 17:20:00,488 INFO: Microblog startup [in C:\Users\Alec\Documents\Development\FlaskStuff\microblog\app\__init__.py:52]
2020-03-17 17:20:14,851 INFO: Microblog startup [in C:\Users\Alec\Documents\Development\FlaskStuff\microblog\app\__init__.py:52]
2020-03-17 17:20:23,204 ERROR: Exception on /translate [POST] [in c:\users\alec\documents\development\flaskstuff\microblog\venv\lib\site-packages\flask\app.py:1890]
Traceback (most recent call last):
  File "c:\users\alec\documents\development\flaskstuff\microblog\venv\lib\site-packages\flask\app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "c:\users\alec\documents\development\flaskstuff\microblog\venv\lib\site-packages\flask\app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "c:\users\alec\documents\development\flaskstuff\microblog\venv\lib\site-packages\flask\app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "c:\users\alec\documents\development\flaskstuff\microblog\venv\lib\site-packages\flask\_compat.py", line 39, in reraise
    raise value
  File "c:\users\alec\documents\development\flaskstuff\microblog\venv\lib\site-packages\flask\app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "c:\users\alec\documents\development\flaskstuff\microblog\venv\lib\site-packages\flask\app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "c:\users\alec\documents\development\flaskstuff\microblog\venv\lib\site-packages\flask_login\utils.py", line 272, in decorated_view
    return func(*args, **kwargs)
  File "C:\Users\Alec\Documents\Development\FlaskStuff\microblog\app\routes.py", line 191, in translate_text
    text_input = data['text']
TypeError: 'NoneType' object is not subscriptable
2020-03-17 17:21:42,934 INFO: Microblog startup [in C:\Users\Alec\Documents\Development\FlaskStuff\microblog\app\__init__.py:52]
2020-03-17 17:21:47,797 ERROR: Exception on /translate [POST] [in c:\users\alec\documents\development\flaskstuff\microblog\venv\lib\site-packages\flask\app.py:1890]
Traceback (most recent call last):
  File "c:\users\alec\documents\development\flaskstuff\microblog\venv\lib\site-packages\flask\app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "c:\users\alec\documents\development\flaskstuff\microblog\venv\lib\site-packages\flask\app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "c:\users\alec\documents\development\flaskstuff\microblog\venv\lib\site-packages\flask\app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "c:\users\alec\documents\development\flaskstuff\microblog\venv\lib\site-packages\flask\_compat.py", line 39, in reraise
    raise value
  File "c:\users\alec\documents\development\flaskstuff\microblog\venv\lib\site-packages\flask\app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "c:\users\alec\documents\development\flaskstuff\microblog\venv\lib\site-packages\flask\app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "c:\users\alec\documents\development\flaskstuff\microblog\venv\lib\site-packages\flask_login\utils.py", line 272, in decorated_view
    return func(*args, **kwargs)
  File "C:\Users\Alec\Documents\Development\FlaskStuff\microblog\app\routes.py", line 191, in translate_text
    text_input = data[0]['translations'][0]['text']
TypeError: 'NoneType' object is not subscriptable
2020-03-17 17:23:07,361 INFO: Microblog startup [in C:\Users\Alec\Documents\Development\FlaskStuff\microblog\app\__init__.py:52]
